nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -130,7 +130,8 @@
 @Tags({"http", "https", "rest", "client"})
 @InputRequirement(Requirement.INPUT_ALLOWED)
 @CapabilityDescription("An HTTP client processor which can interact with a configurable HTTP Endpoint. The destination URL and HTTP Method are configurable."
-        + " FlowFile attributes are converted to HTTP headers and the FlowFile contents are included as the body of the request (if the HTTP Method is PUT, POST or PATCH).")
+        + " When the HTTP Method is PUT, POST or PATCH, the FlowFile contents are included as the body of the request and FlowFile attributes are converted"
+        + " to HTTP headers (optionally, if configured to be sent via the 'Attributes to Send' property)")
```

Comment:
It is worth noting that `Attributes to Send` is now different than the display name `Request Header Attributes Pattern`. Perhaps just generalizing this statement is the best approach.
```suggestion
        + " to HTTP headers based on configuration properties.")
```

Reply:
Oops, I missed that I used `.name` and not `.displayName`. Good catch. Also, I prefer to keep the work "optionally", but made the reference to generic "configuration properties" as suggested.